<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWin::TrackMouseEffectConfigForm</class>
 <widget class="QWidget" name="KWin::TrackMouseEffectConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>104</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::FieldsStayAtSizeHint</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="KTitleWidget" name="ktitlewidget">
     <property name="text">
      <string>Trigger on</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="modifierRadio">
     <property name="text">
      <string>Modifiers</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="kcfg_Alt">
        <property name="text">
         <string>Alt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_Control">
        <property name="text">
         <string>Ctrl</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_Shift">
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_Meta">
        <property name="text">
         <string>Meta</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="shortcutRadio">
     <property name="text">
      <string>Shortcut</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KKeySequenceWidget" name="shortcut">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>kkeysequencewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>shortcutRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>shortcut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifierRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
